.github/workflows/parse_changed_files.py

```diff
@@ -0,0 +1,86 @@
+import re
+import sys
+from pathlib import Path
+from typing import List, Tuple
+
+PLUGIN_DIRS = ['benchmarks', 'data', 'models', 'metrics']
+
+
+def get_changed_files() -> Tuple[List[str], List[str]]:
+    changed_files = sys.argv[1]
+    changed_files_list = changed_files.split()
+
+    plugin_files_changed = []
+    non_plugin_files_changed = []
+
+    for f in changed_files_list:
+        if not any(plugin_dir in f for plugin_dir in PLUGIN_DIRS):
+            non_plugin_files_changed.append(f)
+        else:
+            plugin_files_changed.append(f)
+
+    return plugin_files_changed, non_plugin_files_changed
+
+
+def is_plugin_only(plugins_dict, non_plugin_files_changed):
+    if len(non_plugin_files_changed) > 0:
+        plugins_dict["is_plugin_only"] = "False"
+    else:
+        plugins_dict["is_plugin_only"] = "True"
+
+    return plugins_dict
```

Comment:
```suggestion
def is_plugin_only(plugins_dict: dict, non_plugin_files_changed: List[str]):
    """ 
    Stores `plugins_dict['is_plugin_only']` `"True"` or `"False"` 
    depending on whether there are any changed non-plugin files.
    """
    plugin_only = len(non_plugin_files_changed) > 0
    plugins_dict["is_plugin_only"] = "False" if plugin_only else "True"
```
This might be stylistic, but I have been taught to not return a variable if it's a function parameter that has its internals changed

Reply:
(I know javascript does it differently, but javascript is also not a well-designed language)